lib/iris/experimental/regrid.py

```diff
@@ -1312,3 +1313,401 @@ def regridder(self, src_grid, target_grid):
 
         """
         return _CurvilinearRegridder(src_grid, target_grid, self.weights)
+
+
+class _ProjectedUnstructuredRegridder(object):
```

Comment:
i am interested why this is private.  it is in experimental, I don't think it has to be further obfuscated.  Do you think it ought to stay private?